widgetssdk/src/main/java/com/glia/widgets/di/ControllerFactory.java

```diff
@@ -308,4 +311,11 @@ public EndScreenSharingContract.Controller getEndScreenSharingController() {
     public VisitorCodeContract.Controller getVisitorCodeController() {
         return new VisitorCodeController(dialogController, repositoryFactory.getVisitorCodeRepository(), repositoryFactory.getGliaEngagementRepository());
     }
+
+    public ActivityWatcherContract.Controller getActivityWatcherController() {
+        if (activityWatcherController == null) {
+            activityWatcherController = new ActivityWatcherController(getCallVisualizerController(), getScreenSharingController(), serviceChatHeadController);
```

Comment:
NIT. I would suggest moving each parameter to a separate line.

Reply:
done